third-party/proto-google-common-protos/build.gradle.kts

```diff
@@ -20,5 +20,7 @@ pureKotlin()
 enablePublishing()
 
 dependencies {
-    protobuf(libraries.protoGoogleCommonProtos)
+    protobuf(libraries.protoGoogleCommonProtos) {
+        exclude(group = "com.google.protobuf", module = "protobuf-java")
```

Comment:
maybe pull the exclude into Dependencies.kt

Reply:
Maybe, but one day when we move to version catalogs it'll have to come back.